docs/guides/top/block-io.md

```diff
@@ -5,7 +5,8 @@ description: >
   Visualize the processes generating the most block device I/O
 ---
 
-The `top block-io` gadget is used to trace block devices I/O.
+The `top block-io` gadget is used to visualize the containers generating
```

Comment:
nit: in other guides the gadgets name is not surrounded by '``'

Reply:
Yeah, the original guides were not consistent in this sense. I've removed the inverted quotes around the names of the gadgets throughout.